docs/automation/ROS/ros_p2.md

```diff
@@ -0,0 +1,36 @@
+## ROS Parameters and Parameter Server
+
+Parameter server is a collection of values or parameters that can be retrieved or modified by the nodes during runtime upon requests through command prompt, nodes or launch files. Parameters are intended to be static, globally available values like integers, floats, strings and boolen values and can be stored independently or within a YAML file. Parameters are meant to be gloablly viewable so nodes can easily inspect the configuration state of the system and modify if necessary.
+
+### Accessing and setting Parameters
+#### Via command line 
+Parameters can be accessed, modified or deleted using the `rosparam` command line utility in the `rosbash` suite of terminal commands.
+1) To list all the parameters : <br/> 
+    ```rosparam list```
+    
+   Or to list all the parameters in a specific namespace : <br/> 
```

Comment:
Can you add the ros wiki namespace link too?

Reply:
Cool, I'll add a resource